utils/swift_build_support/swift_build_support/products/earlyswiftdriver.py

```diff
@@ -42,9 +42,9 @@ def is_before_build_script_impl_product(cls):
         return True
 
     def should_build(self, host_target):
-        # Temporarily disable for non-darwin since this build never works
-        # outside of that case currently.
-        if sys.platform != 'darwin':
+        # Temporarily disable for non-darwin and non-Android since this build
+        # never works outside of those two cases currently.
+        if sys.platform != 'darwin' and 'ANDROID_DATA' not in os.environ:
```

Comment:
Still not working on linux, @bnbarham? I could just remove these two lines instead, as it is working well for me natively on Android.

Reply:
I haven't really looked into getting early swift-driver going TBH, so I'm not sure. It could be that it just works now, if you feel like it you could open a PR and run a toolchain build and see what happens 😅

Reply:
OK, can check that separately.

Reply:
Spun off removing this into #69564.